CameraBrowser/CameraBrowserSample/ImageTransformer/CGImageRefToNSImageTransformer.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Foundation;
+using ObjCRuntime;
+using AppKit;
+using CoreGraphics;
+
+namespace CameraBrowserSample
+{
+    [Register("CGImageRefToNSImageTransformer")]
+    public class CGImageRefToNSImageTransformer: NSValueTransformer
+	{
+        [Export("transformedValueClass")]
+        public static new Class TransformedValueClass => new Class(typeof(NSImage));
+
+        [Export("allowsReverseTransformation")]
+        public static new bool AllowsReverseTransformation => false;
+
+        public override NSObject TransformedValue(NSObject value)
+        {
+            if (value == null)
+                return null;
+
+            var img = Runtime.GetINativeObject<CGImage>(value.Handle, false);
```

Comment:
img here can be null. Enable nullability in the project, you will have a number of warnings.